tests/comparisons/Migration/testCompositeConstraintsSnapshot.php

```diff
@@ -34,26 +34,31 @@ public function up()
             ])
             ->addIndex(
                 [
-                    'slug',
-                ],
-                ['unique' => true]
+                    'category_id',
+                ]
+            )
+            ->addIndex(
+                [
+                    'product_id',
+                ]
```

Comment:
That was my initial thought. By at some point, the resulting snapshot was raising exception when migrating.
But that was a few commits ago and I corrected lots of stuff since, maybe there was something else causing trouble.
I'll double check.